src/SentryLogger.php

```diff
@@ -34,13 +34,16 @@ class SentryLogger extends Logger
     /** @var array */
     private $priorityMapping = [];
 
-    public function register(string $dsn, string $environment, float $tracesSampleRate)
+    /** @var float */
+    private $tracesSampleRate = 0.0;
```

Comment:
I've double checked the documentation and the default shouldn't be `0.0`. That would mean that no errors would be sent. This is in the docs:

> To send a representative sample of your errors to Sentry, set the sample_rate option in your SDK configuration to a number between 0 **(0% of errors sent)** and 1 (100% of errors sent).

The default here in the Logger should be `null`.

Reply:
I am not sure if this possible - sentry-php package expecting only int/float variable - [Options.php](https://github.com/getsentry/sentry-php/blob/master/src/Options.php#L803)